README.md

```diff
@@ -49,7 +49,8 @@ See [Troubleshooting Tips](doc/troubleshooting.md) as well as [Yarn tips](doc/ya
 | 0.5.0       | 1.2.0         |
 | 0.5.1       | 1.3.0         |
 | 0.5.2       | 1.3.1         |
-| master      | 1.4.1         |
+| 0.5.3       | 1.4.1         |
```

Comment:
My proposal was to wait to merge this until we release 0.6.0, which will have 1.4.1, and then master will have this and be on 1.5.0

The reason is that it seems wrong to skip Spark versions, from 1.3.x to 1.5.0, as there will be users who want to use 1.4.x in production.  Since job server is more used for prod, I think we can be slightly slower.

> On Sep 19, 2015, at 12:29 PM, Fernando Otero <notifications@github.com> wrote:
> 
> In README.md <https://github.com/spark-jobserver/spark-jobserver/pull/256#discussion_r39922898>:
> 
> > @@ -49,7 +49,8 @@ See [Troubleshooting Tips](doc/troubleshooting.md) as well as [Yarn tips](doc/ya
> >  | 0.5.0       | 1.2.0         |
> >  | 0.5.1       | 1.3.0         |
> >  | 0.5.2       | 1.3.1         |
> > -| master      | 1.4.1         |
> > +| 0.5.3       | 1.4.1         |
> or we can wait for 0.5.3 to be released and then release this one as 0.6.0
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/spark-jobserver/spark-jobserver/pull/256/files#r39922898>.
>